sdk/tests/js/webgl-test-utils.js

```diff
@@ -39,24 +39,30 @@ var loggingOff = function() {
  * @param {number} value The enum value.
  * @return {string} The enum as a string.
  */
-var glEnumToString = function(gl, value) {
-  if (gl.NO_ERROR === undefined)
-    throw new Error('glEnumToString: `gl` must be a WebGL2?RenderingContext');
+var glEnumToString = function(glOrExt, value) {
   if (value === undefined)
     throw new Error('glEnumToString: `value` must not be undefined');
 
-  // Optimization for the most common enum:
-  if (value === gl.NO_ERROR) {
-    return "NO_ERROR";
-  }
-  for (var p in gl) {
-    if (gl[p] == value) {
-      if (p == 'drawingBufferWidth' || p == 'drawingBufferHeight') {
+  if (glOrExt.keyByValue === undefined) {
+    const keyByValue = glOrExt.keyByValue = {};
+
+    for (const k in glOrExt) {
+      if (k == 'drawingBufferWidth' ||
+          k == 'drawingBufferHeight') {
         continue;
       }
-      return p;
+      const v = glOrExt[k];
+      if (keyByValue[v] === undefined) {
+        keyByValue[v] = k;
```

Comment:
I think this is stringifying all of the functions and setting those strings as keys in keyByValue?

Reply:
Poooossibly. The new regex should fix this.